### PR TITLE
[Doc Feature][Assistants API quick start guide][3781093]

### PR DESCRIPTION
**Pull request by v-bvishnu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/assistant-ai-quick-start.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f4b9c2c1b3c99c6163d09cc97ffc0b097a767bc7/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fassistant-ai-quick-start.md) | 91 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c96bd938-5bf8-4c7e-853e-d301780fbe5b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/assistant-ai-quick-start.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e177fd95e6ad8e2b9e42e1f2819f68b75f9254ab/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fassistant-ai-quick-start.md) | 91 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/05723760-47e6-42c0-ab76-e91e6c5b93a4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/assistant-ai-quick-start.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e5f5d0cf42e69cf8b2518bad5c79ce5486233428/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fassistant-ai-quick-start.md) | 88 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dfab38c1-2c90-438f-9834-579ee91d1669) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/assistant-ai-quick-start.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b6df3fdacc3eb37fc4e48244e0f78340e3b3c021/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fassistant-ai-quick-start.md) | 90 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/74add6fe-083d-4d93-86e6-9e42963e90a6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/assistant-ai-quick-start.md](https://github.com/MicrosoftDocs/msteams-docs/blob/44f36bc1dfe253a59117d7ece3d27028c69506f9/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fassistant-ai-quick-start.md) | 90 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/742bb565-57e8-462f-9b5e-6e527be9f659) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/assistant-ai-quick-start.md](https://github.com/MicrosoftDocs/msteams-docs/blob/141966ae7edefe691d27667d9b5467256d789f63/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fassistant-ai-quick-start.md) | 90 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/80fcfc37-30d6-48e3-a2bf-b635e6c53668) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/141966ae7edefe691d27667d9b5467256d789f63/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fad094ff-af7b-486f-8843-4b6cf9acf3b4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/assistants-api-quick-start.md](https://github.com/MicrosoftDocs/msteams-docs/blob/324323d49847ebecd4c8fb732e0dc5f443b0759a/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fassistants-api-quick-start.md) | 90 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5c6f6e43-15ef-420e-91f4-c7ae282bf5fc) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/324323d49847ebecd4c8fb732e0dc5f443b0759a/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bc5eaf94-7cae-4513-90c4-1dc72ae1f12f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/assistants-api-quick-start.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e94f13ac5c20d387274a9caa164910f552fa74c0/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fassistants-api-quick-start.md) | 90 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6d16033b-e65a-4845-b2e4-5d5a14e85c65) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e94f13ac5c20d387274a9caa164910f552fa74c0/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/98d3758e-7c3c-4360-a045-ae43674cae20) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/how-to/Teams conversational AI/assistants-api-quick-start.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+author: surbhigupta
+description: In this module, learn how to quickly try the Assistants API.
+ms.topic: conceptual
+ms.localizationpriority: high
+zone_pivot_groups: assistant-ai-library-quick-start
+ms.author: v-bvishnu
+ms.date: 05/20/2024
+---
+
+# Assistants API quick start guide
```
Comment (aycabas) [inline msteams-platform/bots/how-to/Teams conversational AI/assistants-api-quick-start.md:12]: Should we update the title as "Quick start guide for using Assistants API with Teams AI library" or similar, as Assistants API is an openai / azure product, currently it sounds like it's a quick start guide for another product.

File: msteams-platform/bots/how-to/Teams conversational AI/assistants-api-quick-start.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+Get started with the OpenAI Assistants API using the MathBot sample. This guide uses the OpenAI Code Interpreter tool to help you create an assistant that specializes in mathematics. The bot uses the gpt-3.5-turbo model to chat with Microsoft Teams users and respond in a polite and respectful manner, staying within the scope of the conversation.
```
Comment (aycabas) [inline msteams-platform/bots/how-to/Teams conversational AI/assistants-api-quick-start.md:14]: Get started using OpenAI or Azure OpenAI Assistants API with Teams AI library in MathBot sample.

File: msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md [hunk 3]
```diff
@@ -94,6 +94,12 @@ The bot framework using Teams AI library requires the following:
 
 Action Planner is the main component calling your Large Language Model (LLM) and includes several features to enhance and customize your model. Model plugins simplify configuring your selected LLM to the planner and ships with an OpenAIModel that supports both OpenAI and Azure OpenAI LLMs. Additional plugins for other models like Llama-2 can easily be added, giving you the flexibility to choose what model is best for your use case. An internal feedback loop increases reliability by fixing the subpar responses from the LLM.
 
+## Assistants API
+
+Assistants API allows you to create powerful AI assistants capable of performing a variety of tasks that are difficult to code using traditional methods. It provides programmatic access to OpenAI’s GPT system for tasks ranging from chat to image processing, audio processing, and building custom assistants. The API supports natural language interaction, enabling the development of assistants that can understand and respond in a conversational manner.
```
Comment (aycabas) [inline msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md:99]: Should we include in the end "Teams AI library supports both OpenAI and Azure OpenAI Assistants API (preview) for you to get started building intelligent assistants."

**Review by aycabas — APPROVED** [review]
Added some comments for minor changes, all good in general.

File: msteams-platform/bots/how-to/Teams conversational AI/assistants-api-quick-start.md (hunk 1 above)
Comment (v-bvishnu) [reply msteams-platform/bots/how-to/Teams conversational AI/assistants-api-quick-start.md:12]: Updated.

File: msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md (hunk 3 above)
Comment (v-bvishnu) [reply msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md:99]: Updated.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/assistants-api-quick-start.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0c7d10acb0f535f8ee2f5dde38d43bb09c199122/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fassistants-api-quick-start.md) | 90 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8a20a921-b5f1-4b7e-8285-0c679edf5138) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0c7d10acb0f535f8ee2f5dde38d43bb09c199122/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dd19d383-356f-4527-ad2c-68a2b67e2fc4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/assistants-api-quick-start.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5900a354dbaa59941877d674e60420ce318aa34e/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fassistants-api-quick-start.md) | 90 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8ebe4bd2-e257-4ae2-b7b0-972da5deb3e4) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5900a354dbaa59941877d674e60420ce318aa34e/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d010d2aa-c527-496b-985c-46fa40ced1b3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by govil1811 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/assistants-api-quick-start.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d3be43bc57cf7458d4063f1c377487dc60696897/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fassistants-api-quick-start.md) | 90 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c2f3a477-7e2a-478d-8aa9-a32910dd61f6) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d3be43bc57cf7458d4063f1c377487dc60696897/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/53451db2-36fe-45f0-83ed-1609ad80b248) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by corinagum — APPROVED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/assistants-api-quick-start.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b202f03d884272c3e4a06a22826f4db187e99a84/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fassistants-api-quick-start.md) | 90 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e78cf1d9-3534-46cf-829c-603bdaeff32b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b202f03d884272c3e4a06a22826f4db187e99a84/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b515c755-6501-42da-9c98-74f4227f23d6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/assistants-api-quick-start.md](https://github.com/MicrosoftDocs/msteams-docs/blob/88ac72d71dca2f13701263277eec5cbeac059bf4/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fassistants-api-quick-start.md) | 90 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5f9c9653-eb56-4fad-8fde-d80ffef530c3) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/88ac72d71dca2f13701263277eec5cbeac059bf4/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a9bf3903-e010-4919-a5a1-490e3355197e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Review by v-deabira — APPROVED** [review]

**Pull request merged**